src/server/pfs/server/driver.go

```diff
@@ -498,6 +498,94 @@ func (d *driver) inspectProject(ctx context.Context, project *pfs.Project) (*pfs
 	return pi, nil
 }
 
+func (d *driver) findCommits(ctx context.Context, request *pfs.FindCommitsRequest, cb func(response *pfs.FindCommitsResponse) error) error {
```

Comment:
Is there an advantage to having a callback function here rather than a gRPC stream?

Reply:
I'm not sure actually. I was attempting to stay consistent with the pattern used for streaming RPCs based on our existing implementations in our ListCommit/ListBranch/ListRepo endpoints. They seem to pass in the server.Send() as a callback which is used by the driver to stream back data to the client.